src/Microsoft.Dnx.Runtime.Sources/Impl/DependencyInjection/ActivatorUtilities.cs

```diff
@@ -79,6 +79,9 @@ public static T CreateInstance<T>(IServiceProvider services)
                 ParameterInfo[] parameters = constructors[0].GetParameters();
                 return services =>
                 {
+                    if (services == null)
+                        throw new InvalidOperationException($"Unable to inject into {type.Name} constructor, current host doesn't support dependency injection");
```

Comment:
Suggestion for message: "Cannot create instance of {type} because the current host doesn't support dependency injection"